ui/index.d.ts

```diff
@@ -181,15 +181,20 @@ export declare abstract class AzExtTreeItem {
      *
      * If not provided, an id is generated using the treeItem's label. **Note** that when labels change, ids will change and that selection and expansion state cannot be kept stable anymore.
      */
-    public id?: string;
+    public set id(id: string | undefined)
```

Comment:
Seems like Brandon got it to work by making them abstract instead of forcing people to use accessors

https://github.com/microsoft/vscode-docker/blob/main/src/tree/AzExtTreeItemIntermediate.ts#L10